### PR TITLE
Added new authType parameter in auth messages, handling the config read

### PR DESCRIPTION
**Pull request by karthikbr82**
cleaner way

### CODE REVIEW

File: callstats-java-sdk/src/main/java/io/callstats/sdk/httpclient/CallStatsAsyncHttpClient.java [hunk 1]
```diff
@@ -51,7 +51,15 @@ public CallStatsAsyncHttpClient() {
 			if (input !=null){
 				prop.load(input);
 				BASE_URL = prop.getProperty("CallStats.BaseURL");
```
Comment (marcinag) [inline callstats-java-sdk/src/main/java/io/callstats/sdk/httpclient/CallStatsAsyncHttpClient.java]: If it's not there can we have a crash here?

**Comment by marcinag** [comment]
LGTM

File: callstats-java-sdk/src/main/java/io/callstats/sdk/httpclient/CallStatsAsyncHttpClient.java (hunk 1 above)
Comment (karthikbr82) [inline callstats-java-sdk/src/main/java/io/callstats/sdk/httpclient/CallStatsAsyncHttpClient.java]: No it wont crash, but returns null.

**Comment by vr000m** [comment]
This is blocked on the authentication being merged in  https://github.com/dialogue-io/node-collector

**Pull request merged**